packages/react-native-editor/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ For each user feature we should also add a importance categorization label  to i
 ## 1.52.2
 
 -   [*] Disabled featured image banner on iOS. [#31681]
+-   [***] Audio block now available on WP.com sites on the free plan. [#31966]
```

Comment:
I think this entry should be located in the `Unreleased` section.

Reply:
Indeed! Thanks for spotting. I am gonna keep an eye out for these when I do merges from `trunk` 👍🏾